src/Sulu/Bundle/MediaBundle/Resources/views/Template/media-info.html.twig

```diff
@@ -1,18 +1,30 @@
 <form id="media-info">
     <div class="grid">
         <div class="grid-row">
-            <label for="media-edit-title"><%= translate('public.title') %></label>
-            <input class="form-element" type="text" id="media-edit-title" value=""
-                   data-validation-required="true"
-                   data-mapper-property="title"/>
-        </div>
-        <div class="grid-row">
-            <label for="media-edit-description"><%= translate('public.description') %></label>
-            <textarea class="small noResize form-element" id="media-edit-description"
-                      data-mapper-property="description"></textarea>
+            <% if (!!media.thumbnails && !!media.thumbnails['200x200']) { %>
+            <div class="grid-col-4">
+                <div class="media-edit-preview-image" style="background-image: url('<%=media.thumbnails['200x200']%>');"></div>
```

Comment:
it is not so easy to place an image in the center of the parent container! i dont know how big the image is because if it (the original) is smaller that 200px it comes in a smaller dimension and it should be placed in the center of the container... if we use here an image tag we have to do this with javascript ...